layouts/partials/js.html

```diff
@@ -1,8 +1,16 @@
-<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-<script src="{{ .Site.BaseURL }}js/scrollreveal.min.js"></script>
-<script type='application/javascript' src='{{ .Site.BaseURL }}js/smooth-scrollbar.js'></script>
-<script src="{{ .Site.BaseURL }}js/baffle.min.js"></script>
-<script src="{{ .Site.BaseURL }}js/main.js"></script>
+<script>
+(function () {
+  if (window.fetch) return;
+  document.write('<script src="https://cdn.jsdelivr.net/jquery/3.2.1/jquery.slim.min.js"><\/script>');
```

Comment:
Looks like you may be able to switch to the slim build. Based on my testing everything seems to be working, and it's a bit smaller than the full version, which contains `jQuery.ajax`, `jQuery.animate` and other stuff generally not necessary in modern browsers.